src/_forms.scss

```diff
@@ -60,239 +150,306 @@ category: Common
-    border: 0 none;
-    color: #ffffff;
-    cursor: pointer;
-    display: inline-block;
-    font-size: 14px;
-    margin: 0 $gs-gutter 0 0;
-    min-width: gs-span(2);
-    padding: 11px $gs-gutter/2;
-    outline: none;
-    text-align: center;
+// width needed for iphone so select doesn't exceed screen width
+.select--wide {
+    width: 100%;
+}
 
-    &:hover,
-    &:focus {
-        background: $pasteup-forms-submit-background-focus;
+.select--minimal {
+    border: 0;
+    outline: 0;
+    background-color: #ffffff;
+    font-weight: bold;
+}
+
+// Psuedo radio form inputs
+// ============================================================================
+
+.pseudo-radio {
+    @include box-sizing(border-box);
+    width: 100%;
+    border: $pasteup-forms-radio-border;
+    padding: rem($gs-baseline) rem($gs-gutter) rem($gs-baseline) rem(36px);
+    position: relative;
+
+    &:hover {
+        border: $pasteup-forms-radio-border-hover;
     }
 
-    &:active {
-        background: $pasteup-forms-submit-background-active;
+    &:before {
+        @include radio-circle($pasteup-forms-radio-colour);
     }
 }
 
-.submit-input[disabled] {
-    background: $pasteup-forms-submit-background-disabled;
+.pseudo-radio-input {
+    display: none;
+
+    &:checked + .pseudo-radio {
+        border: $pasteup-forms-radio-border-checked;
+
+        &:before {
+            background-color: $pasteup-forms-radio-colour-checked;
+        }
+    }
 }
 
-.check-label,
-.radio-label {
-    display: block;
-    font-family: $pasteup-forms-fields-font-family;
-    font-size: 14px;
-    margin-bottom: $gs-baseline/3;
-    padding-left: $gs-gutter;
+.pseudo-radio__label {
+    @include fs-bodyCopy(2);
+    line-height: 1.2;
 }
 
-.check-label--helper {
-    display: inline-block;
-    vertical-align: middle;
+.pseudo-radio__header {
+    @include fs-bodyHeading(2);
+    line-height: 1.2;
+    margin-bottom: rem($gs-baseline / 2);
 }
 
-[type=checkbox],
-[type=radio] {
+.pseudo-radio__note {
+    @include fs-textSans(3);
+    margin: 0;
+}
+
+// Cvc
+// ============================================================================
+
+.cvc {
+    font-family: $f-sans-serif-text;
+    padding: rem($gs-baseline / 2) rem($gs-baseline / 2) rem($gs-baseline / 2) 0;
+    font-size: rem(13px);
+    line-height: rem(16px);
+}
+
+.cvc__image {
     float: left;
-    height: 13px;
-    margin-left: -$gs-gutter;
-    margin-top: 2px;
-    width: 13px;
 }
 
+.cvc__detail {
+    margin-left: rem(70px);
+}
+
+.cvc__heading {
+    color: $pasteup-forms-label-colour;
+    font-weight: bold;
+    font-family: $f-sans-serif-text;
+    font-size: rem(13px);
+    line-height: rem(16px);
+    margin: 0;
+}
+
+.cvc__body {
+    color: $pasteup-forms-label-colour;
+    margin-top: 0;
+    margin-bottom: rem(16px);
+}
+
+
+// Credit card notes
+// ============================================================================
+
+.credit-card-note {
```

Comment:
Like CVC styles this is still a little awkward, can we remove for now if you don't need it yet?